db/patterns/naver.com.eno

```diff
@@ -11,6 +11,7 @@ naver.net
 --- filters
 ||search.naver.com^$3p
 ||wcs.naver.net/wcslog.js
+||siape.veta.naver.com
```

Comment:
Do you think it would be sufficient to block the domain only for third-party requests? In other words, `||siape.veta.naver.com^$3p`

Reply:
Unlike `adcr.naver.com`, `siape.veta.naver.com` doesn't cause any breakage. However, I think it's good to keep in `$3p`. :)